### DISCUSSION TITLE
Show filename in server comments for badge/game icon changes

### DISCUSSION THREAD

**Discussion by televandalist** [RetroAchievements/RAWeb#1063] [Ideas]
`28 Jun 2022 22:51 televandalist changed the game icon`

`28 Jun 2022 22:51 televandalist changed the game icon from 012345.png to 012346.png`

**Comment by Tsearo** [comment]
Seems like it would be simple to do but I'm curious what the benefit would be for this, could you elaborate a use case?

**Reply by MrOwnership** [reply]
Images are not deleted from the server when new ones are uploaded. If something needs to be reverted then the image can easily be downloaded and reuploaded. It would be easier to see a history of previous images.

Might be worth adding in for boxart, screenshots in into #1054 for achievements badge uploads as well.

**Comment by Jamiras** [comment]
This information is now tracked (if the images are updated from the management app - #3039).

Note that it shows the image though.
![image](https://github.com/user-attachments/assets/248e58ed-c167-4501-b502-284ad8c539b2)

Maybe we can add a tooltip to see the filename? You can get it using the Inpsect browser tool.
